modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -174,14 +174,14 @@ const TooltipProvider = () => {
 
 		const TOOLTIP_ENTER = delegate(
 			document.body,
-			'mouseenter',
+			'mouseover',
 			SELECTOR_TOOLTIP,
 			() => dispatch({target: state.target, type: 'show'})
 		);
 
 		const TOOLTIP_LEAVE = delegate(
 			document.body,
-			'mouseleave',
+			'mouseout',
```

Comment:
Slightly scary, but I think it makes sense. `mouseover` will fire on entering any child of `document.body`, whereas `mouseenter` would oly fire for `document.body` itself.

Reply:
That's exactly what I saw when debugging, as you said `mouseenter` will fire only when entering  `document.body` and not it's children.